pkg/framework/plugins/nodeutilization/nodeutilization.go

```diff
@@ -290,8 +279,8 @@ func evictPodsFromSourceNodes(
 		klog.V(1).InfoS("Evicting pods based on priority, if they have same priority, they'll be evicted based on QoS tiers")
 		// sort the evictable Pods based on priority. This also sorts them based on QoS. If there are multiple pods with same priority, they are sorted based on QoS tiers.
 		podutil.SortPodsBasedOnPriorityLowToHigh(removablePods)
-		evictPods(ctx, removablePods, node, totalAvailableUsage, taintsOfDestinationNodes, podEvictor, strategy, continueEviction)
-		klog.V(1).InfoS("Evicted pods from node", "node", klog.KObj(node.node), "evictedPods", podEvictor.NodeEvicted(node.node), "usage", node.usage)
```

Comment:
Has this line got accidentally removed?

Reply:
no, that is the reason why i use `podEvictor ` parameter.
because it implement the NodeEvicted, but `d.handle.Evictor()` not implement.

Reply:
is it is ok for you, can i click resolve conversation?

Reply:
Make sense. The framework is expected to log the number of evicted pods for a plugin/extension point. Once implemented, this line would go away.